prometheus_metrics/Cargo.toml

```diff
@@ -8,7 +8,8 @@ authors = ["Grandine <info@grandine.io>"]
 anyhow = { workspace = true }
 features = { workspace = true }
 futures = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order